app/design/frontend/base/default/template/boltpay/replace.phtml

```diff
@@ -80,21 +80,29 @@ $additionalClasses = $this->boltHelper()->getAdditionalButtonClasses();
 
                                 var element = elements[i];
                                 found_elements = true;
-
-                                element.style.display = "none";
-
-                                var bolt_button =  document.createElement("div");
-                                bolt_button.setAttribute("class",selectors[selector]);
-
-                                <?php $buttonColor = $this->boltHelper()->getBoltPrimaryColor();?>
-                                <?php if($buttonColor):?>
-                                    bolt_button.setAttribute("style", "--bolt-primary-action-color:<?php echo $buttonColor?>");
-                                <?php endif; ?>
-
-                                element.parentNode.insertBefore(bolt_button, element);
-
-                                if (element.parentNode.style.display === "") {
-                                    element.parentNode.style.display = "block";
+                                /**
+                                 * Escape double replace buttons
+                                **/
+                                var bolt = element.getAttribute("bolt");
```

Comment:
Can we use better variable names and also use the modern HTML `data-` attributes?
```
var wasReplaced = element.getAttribute('data-was-replaced');

if ('wasReplaced' !== 'true') {
   ...
   element.setAttribute("data-was-replaced", "true");
    
```

This way we can have both readable and valid HTML instead of this HTML which will not validate.